frontend/js/src/settings/import/ImportListens.tsx

```diff
@@ -49,47 +49,35 @@ export default function Import() {
         </div>
       )}
       <p>
-        Most users will want to import from Last.fm directly.
-        <br />
+        Import your existing listen history from other databasess. To submit <em>new</em>
+        listens, please visit the{" "} <a href="https://listenbrainz.org/add-data/">Connect services</a>
+        and{" "} <a href="https://listenbrainz.org/settings/music-services/details/">Submitting data</a>
+        pages.
+      </br>
```

Comment:
These are almost good, but they're making the linters freak out!
Linters are tools that check the code has no issues.

In this case, it's pointing out below that it's expecting closing tags and whatnot, but the real issue is this: self-closing HTML tags need the slash to be at the end like so:
```suggestion
      <br/>
```